var/spack/repos/builtin/packages/texlive/package.py

```diff
@@ -86,52 +71,34 @@ class Texlive(AutotoolsPackage):
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 
-    # The following variant is only for the "live" binary installation.
-    # There does not seem to be a complete list of schemes.
-    # Examples include:
-    #   full scheme (everything)
-    #   medium scheme (small + more packages and languages)
-    #   small scheme (basic + xetex, metapost, a few languages)
-    #   basic scheme (plain and latex)
-    #   minimal scheme (plain only)
-    # See:
-    # https://www.tug.org/texlive/doc/texlive-en/texlive-en.html#x1-25025r6
-    variant(
-        "scheme",
-        default="small",
-        values=("minimal", "basic", "small", "medium", "full"),
-        description='Package subset to install, only meaningful for "live" ' "version",
-    )
-
-    depends_on("perl", type="build", when="@live")
-    depends_on("pkgconfig", when="@2019:", type="build")
-
-    depends_on("cairo+X", when="@2019:")
-    depends_on("freetype", when="@2019:")
-    depends_on("ghostscript", when="@2019:")
-    depends_on("gmp", when="@2019:")
-    depends_on("harfbuzz+graphite2", when="@2019:")
-    depends_on("icu4c", when="@2019:")
-    depends_on("libgd", when="@2019:")
-    depends_on("libpaper", when="@2019:")
-    depends_on("libpng", when="@2019:")
-    depends_on("libxaw", when="@2019:")
-    depends_on("libxt", when="@2019:")
-    depends_on("mpfr", when="@2019:")
-    depends_on("perl", when="@2019:")
-    depends_on("pixman", when="@2019:")
-    depends_on("poppler@:0.84", when="@2019:")
-    depends_on("teckit", when="@2019:")
-    depends_on("zlib-api", when="@2019:")
-    depends_on("zziplib", when="@2019:")
+    depends_on("pkgconfig", type="build")
+
+    depends_on("cairo+X")
+    depends_on("freetype")
+    depends_on("ghostscript")
+    depends_on("gmp")
+    depends_on("harfbuzz+graphite2")
+    depends_on("icu4c")
+    depends_on("libgd")
+    depends_on("libpaper")
+    depends_on("libpng")
+    depends_on("libxaw")
+    depends_on("libxt")
+    depends_on("mpfr")
+    depends_on("perl")
+    depends_on("pixman")
+    depends_on("poppler@:0.84", when="@:2019")
+    depends_on("poppler")
```

Comment:
2020 may be the last version that actually depends on poppler, see https://github.com/TeX-Live/texlive-source/blob/e888cd260121c19da9d6f294c9e03f35c05b40d3/m4/kpse-xpdf-flags.m4#L15 and https://github.com/TeX-Live/texlive-source/commit/ef05ec36803a62e5557ee2c6d107802ad544f193.

Reply:
In my local installation (from the system) poppler is not needed so this seems to be true. I changed it to only apply until 2020.